unison-src/transcripts/merges.output.md

```diff
@@ -120,13 +120,13 @@ We can also delete the fork if we're done with it. (Don't worry, it's still in t
   Note: The most recent namespace hash is immediately below this
         message.
   
-  ⊙ #qb1es1ffsc
+  ⊙ #6s4tc161c3
```

Comment:
The hash of root namespace changed because one of the definitions in `IOSource` is a block which is parsed into something different than before. @aryairani what if anything do we need to do about this?

Reply:
Let's get rid of it, per #1056.  In the meantime, it will only affect people who `use .builtin` with a codebase initialized after this PR.